checksec

```diff
@@ -175,38 +168,48 @@ echo_message() {
       echo -n -e "$3"
   elif [[ $format == "json" ]]; then
       echo -n -e "$4"
-  else #default to cli 
+  else #default to cli
       echo -n -e "$1"
   fi
 }
 
 # check selinux status
 getsestatus() {
   $debug && echo -e "\n***fuction getsestatus"
-if (command_exists getenforce); then
-  $debug && echo "***fuction getsestatus->getenforce"
-  sestatus=$(getenforce)
-  if [ "$sestatus" == "Disabled" ]; then
-    status=0
-  elif [ "$sestatus" == "Permissive" ]; then
-    status=1
-  elif [ "$sestatus" == "Enforcing" ]; then
-    status=2
-  fi
-elif (command_exists sestatus); then
-  $debug && echo "***fuction getsestatus->sestatus"
-  sestatus=$(sestatus | grep "SELinux status" | awk '{ print $3}')
-  if [ "$sestatus" == "disabled" ]; then
-    status=0
-  elif [ "$sestatus" == "enabled" ]; then
-    sestatus2=$(sestatus | grep "Current" | awk '{ print $3}')
-    if [ "$sestatus2" == "permissive" ]; then
+  if (command_exists getenforce); then
+    $debug && echo "***fuction getsestatus->getenforce"
+    sestatus=$(getenforce)
+    if [ "$sestatus" == "Disabled" ]; then
+      status=0
+    elif [ "$sestatus" == "Permissive" ]; then
+      status=1
+    elif [ "$sestatus" == "Enforcing" ]; then
+      status=2
+    fi
+  elif (command_exists /usr/sbin/getenforce); then
+    $debug && echo "***fuction getsestatus->/usr/sbin/getenforce"
+    sestatus=$(/usr/sbin/getenforce)
+    if [ "$sestatus" == "Disabled" ]; then
+      status=0
+    elif [ "$sestatus" == "Permissive" ]; then
```

Comment:
when running as non-root, /usr/sbin is not in the path list, so getenforce is not found (also it works as non-root)